addons/website/static/src/interactions/dropdown/dropdown_mega_menu.js

```diff
@@ -1,7 +1,7 @@
 import { Interaction } from "@website/core/interaction";
 import { registry } from "@web/core/registry";
 
-class MegaMenuDropdown extends Interaction {
+class DropdownMegaMenu extends Interaction {
```

Comment:
love this change :)

Reply:
It is to be more consistent with the naming and organize the files.
Obviously file names can be changed later